code/project_euler_problem19.java

```diff
@@ -0,0 +1,90 @@
+import java.util.HashMap;
```

Comment:
This file is in the wrong place, and has the wrong name. Look at the other project eulers as an example.

Reply:
@arnavb  should I make new pull request under that folder.

Reply:
No, you can just edit the file name, see [this](https://blog.github.com/2013-03-15-moving-and-renaming-files-on-github/) @Awani-Shukla